shell/platform/tizen/external_texture_surface_gl_tizen.cc

```diff
@@ -14,9 +16,19 @@ EVAS_GL_GLOBAL_GLES3_DECLARE();
 #include <EGL/eglext.h>
 #include <GLES2/gl2ext.h>
 #include <GLES3/gl32.h>
-#endif
 
-#include <tbm_surface.h>
+#include <tbm_bufmgr.h>
+#include <tbm_surface_internal.h>
+#ifndef EGL_DMA_BUF_PLANE3_FD_EXT
+#define EGL_DMA_BUF_PLANE3_FD_EXT 0x3440
+#endif
+#ifndef EGL_DMA_BUF_PLANE3_OFFSET_EXT
+#define EGL_DMA_BUF_PLANE3_OFFSET_EXT 0x3441
+#endif
+#ifndef EGL_DMA_BUF_PLANE3_PITCH_EXT
+#define EGL_DMA_BUF_PLANE3_PITCH_EXT 0x3442
+#endif
+#endif
```

Comment:
Are these defines necessary?

Reply:
In my case, I can't build without it... can you??

Reply:
I couldn't find any other way :(